### PR TITLE
Added lexic, syntax and tokens for lab1b

### PR DESCRIPTION
**Pull request by livcristi**

### CODE REVIEW

File: Lab 1b/tokens.in [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+Token_list	code
+identifier	0
```
Comment (carlatodoran1507) [inline Lab 1b/tokens.in]: I don't think identifier should be part of the list, unless it's a reserved word

File: Lab 1b/tokens.in [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+other 		9
+while 		10
+loop 		11
+read 		12
+print		13
+{			14
+}			15
+(			16
+)			17
+<			18
```
Comment (carlatodoran1507) [inline Lab 1b/tokens.in]: You also need to add the square brackets here [ ]

File: Lab 1b/tokens.in [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+>			19
+<=			20
+>=			21
+==			22
+!=			23
```
Comment (carlatodoran1507) [inline Lab 1b/tokens.in]: Also add the attribution operator <-

File: Lab 1b/Lexic.txt [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+Specification
+
+Alphabet:
+	a. Lowercase (a-z) and uppercase letters (A-Z) of the English alphabet
+	b. Decimal digits (0-9)
+	c. Special characters !%*()-_+=/<>{}[]:;'"
+
+1. Lexic
+	a. Special symbols, representing:
+		- operators <- ++ + - * / mod < > <= == >= != and or
```
Comment (carlatodoran1507) [inline Lab 1b/Lexic.txt]: Just a question, you have ++, but not - -?

File: Lab 1b/Lexic.txt [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+			special_symbol = "_" | "+" | "-" | "*" | "/" | ":".
+
+			constchar = 'char'.
+
+			char = letter|digit|special_symbol.
+		
+		5. string:
+			
+			string = """{char}""".
+
```
Comment (carlatodoran1507) [inline Lab 1b/Lexic.txt:48]: Why the triple "?

File: Lab 1b/tokens.in (hunk 3 above)
Comment (carlatodoran1507) [reply Lab 1b/tokens.in]: And ++

**Review by carlatodoran1507 — CHANGES_REQUESTED** [review]

File: Lab 1b/Lexic.txt (hunk 4 above)
Comment (livcristi) [reply Lab 1b/Lexic.txt]: Oh, I forgot about it

File: Lab 1b/Lexic.txt (hunk 5 above)
Comment (livcristi) [reply Lab 1b/Lexic.txt:48]: " is a terminal, so it needs to be written between " " (so that makes it " " ")

**Pull request merged**